src/upload.ts

```diff
@@ -281,16 +296,22 @@ export async function uploadLargeFileRequest(
   let splitSizeIntoParts:
     | ((totalSize: number, partCount: number) => Array<{ start: number; end: number }>)
     | undefined = undefined;
+  const staggerPercent = opts.staggerPercent;
```

Comment:
nit: We don't need this.

Reply:
Ended up doing a slight refactor here because if we end up using `parallelUploads: 1` then this has to be set to null.